packages/medusa/src/loaders/feature-flags/order-editing.ts

```diff
@@ -2,9 +2,9 @@ import { FlagSettings } from "../../types/feature-flags"
 
 const OrderEditingFeatureFlag: FlagSettings = {
   key: "order_editing",
-  default_val: false,
+  default_val: true,
   env_key: "MEDUSA_FF_ORDER_EDITING",
   description: "[WIP] Enable the order editing feature",
 }
 
-export default OrderEditingFeatureFlag
+export default OrderEditingFeatureFlag
```

Comment:
since this isn't used anywhere, should we delete it?

Reply:
Wanted to keep it in until admin has been cleaned up. Otherwise, you won't be able to use the UI for it

Reply:
ahh yes, forgot about the admin :D